### PR TITLE
qemu.tests: Add more debug log to check

### PR DESCRIPTION
**Pull request by spcui**
ID: 1231011

### CODE REVIEW

**Comment by spcui** [comment]
Related: https://github.com/autotest/virt-test/pull/2220

**Comment by CongSmile** [comment]
Hi @spcui 

According to the request, we also need some description for the case.

Thanks.

**Comment by xutian** [comment]
@spcui 

Any update for this PR?

**Comment by spcui** [comment]
@CongSmile 
The descriptions are clearly, so not need to update.

**Comment by suqinhuang** [comment]
Hi scui,
steps are not recored in error.context, could you pls update them as well.

Thanks
Suqin

**Comment by suqinhuang** [comment]
Acked-by: Suqin Huang

**Comment by vipmike007** [comment]
LGTM.
Acked-By: Mike Cao \<bcao@redhat.com\>

File: qemu/tests/physical_resources_check.py [hunk 1]
```diff
@@ -313,12 +317,12 @@ def verify_machine_type():
             _ = ("Images check failed with %s errors, check the log for "
```
Comment (xutian) [inline qemu/tests/physical_resources_check.py]: Hi @spcui, 

Please update  '_+= ' too. thanks

Comment (spcui) [inline qemu/tests/physical_resources_check.py]: Hi @xutian 
Should not update to  '_+= ', only print the failed log.

File: qemu/tests/physical_resources_check.py [hunk 2]
```diff
@@ -370,6 +374,6 @@ def verify_machine_type():
         session.close()
         raise error.TestFail("Physical resources check test "
                              "reported %s failures:\n%s" %
-                             (len(n_fail), "\n".join(n_fail)))
+                             (len(n_fail), n_fail))
```
Comment (ypu) [inline qemu/tests/physical_resources_check.py]: Hi the original output seems more readable.

**Comment by spcui** [comment]
Hi @ypu updated, could you help review again? Thanks

**Comment by xutian** [comment]
@spcui  Please update your PR.   '_' used to throwaway as var,  if you still use it again, please use a meaningful var name instead '_'.

**Comment by spcui** [comment]
@xutian updated with your comment, help reivew it.

**Comment by xutian** [comment]
@spcui function check_num return expect_num, but check code not found where use it, so I think return fail_log is enough.

Thanks,
Xu

**Comment by spcui** [comment]
@xutian ok, updated

**Comment by xutian** [comment]
Merged, thanks @spcui

**Pull request merged**